frontend/src/components/auth/TeacherSignup/steps/TeacherSignUpOne.tsx

```diff
@@ -158,7 +158,8 @@ const TeacherSignupOne = ({
       <NavigationButtons
         onContinueClick={onContinueClick}
         onBackClick={() => history.push(TEACHER_LOGIN)}
-        firstPage
+        continueButtonText="Continue"
+        backButtonText="Back to login page"
       />
     </>
   );
```

Comment:
This is totally beyond the scope of your PR and kind of a nit, but we should probably name these components something more expressive than `TeacherSignupOne` to `TeacherSignupFour` LOL